docs/03-extensions/magento1/how-to/setup-magento-extension.mdx

```diff
@@ -0,0 +1,295 @@
+     `app/code/community/Clockworkgeek/Extrarestful` directory
+
+- Enterprise edition version:
+
+  1. Clone or download
+     https://gitlab.blackswift.cloud/front-commerce/magento1-module-front-commerce.git
+  2. Copy `app` directory and paste it in your Magento's root directory
+  3. Clone or download
+     https://gitlab.blackswift.cloud/front-commerce/magento1-module-enterprise-front-commerce
+  4. Copy `app` directory and paste it in your Magento's root directory
+  5. Clone or download https://github.com/PH2M/Magento-Extra-RESTful
+  6. Copy `modules/Clockworkgeek_Extrarestful.xml` file and paste it in your
+     Magento's `app/etc/modules` directory
+  7. Create `app/code/community/Clockworkgeek/Extrarestful` directory in your
+     Magento's
+  8. Copy all directories inside `code` in your Magento's
+     `app/code/community/Clockworkgeek/Extrarestful` directory
+
+## Configuration
+
+:::note
+
+If the installation is successful, in Magento's administration panel, you will
+have a new entry "Front-Commerce" in the top menu and a new tab "Front-Commerce"
+in System > Configuration.
+
+:::
+
+### Check install
+
+Go to the `Front-Commerce > Configuration` admin menu entry. You should see an
+"Installer checker" that will ensure that everything is configured correctly in
+your shop.
+
+Right after your first installation, most of the checks should be invalid.
+Please refer to the next steps to validate them.
+
+### REST roles
+
+Front-Commerce requires you to define 3 roles.
+
+Here are the main steps:
+
+- Go to admin menu entry `System > Web services > REST Roles`
+- You need to have 3 roles, `Guest`, `Customer`, and `Admin`. If you don't,
+  create them.
+- Set all roles access to all resources (Role API Resources tab > Resource
+  Access "All").
+
+See the
+[official documentation](https://docs.magento.com/m1/ce/user_guide/system-operations/web-services-activate.html)
+for detailed information about how to achieve this with Magento.
+
+### REST attributes
+
+Front-Commerce roles must have access to several data on each entities, so it
+can expose them in GraphQL. You must allow each role to read these information.
+
+- Go to admin menu entry `System > Web services > REST Attributes`
+- You can see 3 user types `Guest`, `Customer`, and `Admin`.
+- Set all ACL attributes rules to all resource access (ACL Attribute Rules tab >
+  Resource Access "All").
+
+### REST OAuth Consumer
+
+Front-Commerce must be able to create tokens for users when they log-in. You
+should ensure there is a `Front-Commerce` consumer configured in your install.
+
+- Go to admin menu entry `System > Web services > REST OAuth Consumers`
+- Add New OAuth Consumer:
+  - Name: `Front-Commerce`
+  - Callback URL: `http://local.host` <- is useless but can't be empty
+
+The Key/Secret values should be used to configure your Front-Commerce, with the
+`FRONT_COMMERCE_MAGENTO_CONSUMER_KEY` and
+`FRONT_COMMERCE_MAGENTO_CONSUMER_SECRET` environment variables in your `.env`
```

Comment:
are these still correct? should it not be `FRONT_COMMERCE_MAGENTO1_`

```suggestion
`FRONT_COMMERCE_MAGENTO1_CONSUMER_KEY` and
`FRONT_COMMERCE_MAGENTO1_CONSUMER_SECRET` environment variables in your `.env`
```

**suggestion:** do a global search for  `FRONT_COMMERCE_MAGENTO_` to fix all cases where it has not been updated in v3 docs

Reply:
fixed